docs/site/Loopback-component-authentication.md

```diff
@@ -608,6 +608,23 @@ export class MyApplication extends BootMixin(
 }
 ```
 
+### Use passport based strategies
```

Comment:
> Use passport based strategies 

I believe the standard is that section titles have to be in uppercase

Using Passport-based Strategies